splitio/client/client.py

```diff
@@ -377,7 +377,7 @@ def track(self, key, traffic_type, event_type, value=None, properties=None):
         start = get_current_epoch_time_ms()
         key = input_validator.validate_track_key(key)
         event_type = input_validator.validate_event_type(event_type)
-        should_validate_existance = self.ready and self._factory._apikey != 'localhost'  # pylint: disable=protected-access
+        should_validate_existance = self.ready and self._factory._sdkkey != 'localhost'  # pylint: disable=protected-access
```

Comment:
Since #364 uses `sdk_key` maybe we can update this one for consistency.

Reply:
Yep, updating it now.